doc/manuals/devel/mongoBackend.md

```diff
@@ -186,7 +186,7 @@ Regarding the strategy used in `processContextAttributeVector()` to implement en
 * `toPush`: attributes that need to be added to the entity `attrsName` field in the database (list of attribute names), using the [`$addToSet`](https://docs.mongodb.com/manual/reference/operator/update/addToSet) and [`$each`](https://docs.mongodb.com/manual/reference/operator/update/each) operators.
 * `toPull`: attributes that need to be removed from the `attrsName` field in the database (list of attribute names), using the [`$pullAll` operator](https://docs.mongodb.com/manual/reference/operator/update/pullAll).
 * `locAttr` and `geoJson` are related to modifications in the geolocation information associated to the entity (entity `location` field in the database).
-
+* `dateExpiration` and `replaceDateExpiration` are related to modifications in the TTL expiration date information associated to a transient entity (entity `expDate` field in the database).

```

Comment:
Good addition to the devel manual, but a blank line is  needed just after this one. Otherwise, the text "The update is base [...]" appears in the same paragraph.

Just compare the render on [master](https://github.com/arigliano/fiware-orion/blob/master/doc/manuals/devel/mongoBackend.md#mongoupdatecontext-sr-and-mongonotifycontext-sr) and [your branch](https://github.com/arigliano/fiware-orion/blob/fe0bab47d90d0f9b7f84098348a93b8d1fe2bc3b/doc/manuals/devel/mongoBackend.md) to see what I mean.

Reply:
Fixed in 9b3853d